feature/home/presentation/src/main/java/com/sanaa/presentation/screen/trendingMediaScreen/trendingMoviesScreen/TrendingMoviesScreen.kt

```diff
@@ -92,7 +93,8 @@ fun TrendingMoviesScreen(
         )
 
         NovixAnimatedSnackBarHost(
-            data = snack, onDismiss = { snack = null }
+            data = snack,
+            onDismiss = { snack = null }
         )
 
         if (state.value.showSaveToListBottomSheet && state.value.selectedMediaId != null) {
```

Comment:
Good Job Zainab, 

But I have suggestion, if you can destruct values instead call state.value every time